python/test/unit/language/assert_helper.py

```diff
@@ -148,7 +148,5 @@ def test_assert_nested(caller: str, callee: str):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) == 3:
-        test_assert_nested(sys.argv[1], sys.argv[2])
-    else:
-        test_assert(sys.argv[1])
+    fn = globals()[sys.argv[1]]
+    fn(*sys.argv[2:])
```

Comment:
This approach is more flexible -- we don't need to carefully manage the tests to ensure each one has a unique number of arguments.